WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/EditAttributesViewController.swift

```diff
@@ -15,7 +15,11 @@ final class EditAttributesViewController: UIViewController {
 
     /// Assign this closure to be notified after a variation is created.
     ///
-    var onVariationCreation: ((ProductVariation) -> Void)?
+    var onVariationCreation: ((Product) -> Void)?
+
+    /// Assign this closure to be notified after an attribute  is created.
+    ///
+    var onAttributeCreation: ((Product) -> Void)?
```

Comment:
non-blocking thought: it might be nice having a Combine publisher (observable) for `Product` and passing it along the way for subscription on changes 😄 or keeping the closure, but for any product changes like `var onProductUpdate: ((Product) -> Void)?`

Reply:
I kind of need them to be different, because the consumer `ProductVariationsViewController.navigateToEditAttributeViewController()` perform different actions when an attribute or variations are created.

Reply:
I could have used combine 🤔 but for these simpler callbacks, I didn't see much benefit 😅